assets/src/domain/index.ts

```diff
@@ -5,8 +5,12 @@ const eventEspressoData: EventEspressoData = window?.eventEspressoData;
 const domain = eventEspressoData?.domain;
 
 if (domain) {
-	// todo remove this at some point in the future
-	console.log(`%c importing: /domain/${domain}/entryPoint.ts`, 'color: SkyBlue;');
+	// todo remove console.log at some point in the future
+	console.log(
+		`%c importing: /domain/${domain}/entryPoint.ts with eventEspressoData: `,
+		'color: SkyBlue;',
+		eventEspressoData
```

Comment:
This is sometimes huge for console log if i18n data exists for a non English language. I tested it for French, the translation data was around 800kb.

Reply:
oh ya that's just temp... we'll remove that before shipping